packages/utils/types/tslint.json

```diff
@@ -2,6 +2,7 @@
   "extends": "@definitelytyped/dtslint/dtslint.json",
   "rules": {
     "array-type": [true, "generic"],
-    "semicolon": false
+    "semicolon": false,
+    "no-redundant-jsdoc": false
```

Comment:
I tried all of these options but didn't work so I had to do this instead https://palantir.github.io/tslint/usage/rule-flags/